runner/execer/execers/done.go

```diff
@@ -3,26 +3,32 @@ package execers
 import "github.com/scootdev/scoot/runner/execer"
 
 // Creates a new doneExecer.
-func NewDoneExecer() execer.Execer {
-	return &doneExecer{}
+func NewDoneExecer() *DoneExecer {
+	return &DoneExecer{
+		State: execer.COMPLETE,
```

Comment:
Because it's needed in stateful_scheduler_test.go, in the test TaskGetsMarkedCompletedAfterMaxRetriesFailedRuns, where we want to make sure that a failed run (on the worker) gets considered an error in the scheduler.